recipes/date/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+if(DATE_HEADER_ONLY)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE DATE_HEADER_ONLY)
+endif()
```

```suggestion
conan_basic_setup(TARGETS)

add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```
You can re-use the test_package/CMakeLists.txt instead, so don't need to duplicate content: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package